daffodil-japi/src/test/java/edu/illinois/ncsa/daffodil/example/TestJavaAPI.java

```diff
@@ -58,803 +58,806 @@
 import edu.illinois.ncsa.daffodil.japi.ProcessorFactory;
 import edu.illinois.ncsa.daffodil.japi.UnparseResult;
 import edu.illinois.ncsa.daffodil.japi.ValidationMode;
+import edu.illinois.ncsa.daffodil.japi.infoset.JDOMInfosetInputter;
+import edu.illinois.ncsa.daffodil.japi.infoset.JDOMInfosetOutputter;
 import edu.illinois.ncsa.daffodil.japi.logger.ConsoleLogWriter;
 import edu.illinois.ncsa.daffodil.japi.logger.LogLevel;
-import edu.illinois.ncsa.daffodil.japi.infoset.JDOMInfosetOutputter;
-import edu.illinois.ncsa.daffodil.japi.infoset.JDOMInfosetInputter;
 
 public class TestJavaAPI {
 
-	public java.io.File getResource(String resPath) {
```

Comment:
Looks like all whitespace changes. There is one line of change below. But probably eclipse auto-save settings messed with the whitespace here, so need to be checked, and if possible revised so that it's not going to reindent our work from the way it was already indented.